### DISCUSSION TITLE
"spacy init vectors" fails with AttributeError

### DISCUSSION THREAD

**Discussion by GBR-613** [explosion/spaCy#9586] [Help: Installation] [answered]
## How to reproduce the behaviour
I need to load vectors from a file in word2vec format (named glove.6B.50d.v2w in my example).
I run the following command: 
`python -m spacy init vectors en glove.6B.50d.v2w .`
Result: a failure with the following message: 
`AttributeError: module 'tensorflow.tools.docs.doc_controls' has no attribute 'inheritable_header'`

(The full traceback will be attached in a separate file, as well as my glove.6B.50d.v2w.

### Info about spaCy

- **spaCy version:** 3.1.3
- **Platform:** Windows-10-10.0.19041-SP0
- **Python version:** 3.7.9

### Tensorflow related packages:
- **tensorboard==2.7.0
- **tensorboard-data-server==0.6.1
- **tensorboard-plugin-wit==1.8.0
- **tensorflow==2.6.0
- **tensorflow-estimator==2.7.0
- **tensorflow-hub==0.12.0

**Comment by GBR-613** [comment]
[traceback.txt](https://github.com/explosion/spaCy/files/7453677/traceback.txt)

**Comment by GBR-613** [comment]
It looks like is too large for Github.  I zipped it and made it available from the following link:

[https://drive.google.com/file/d/1BQEHqOsv9BnafEPjDYdAMcOeOxTfNlZk/view?usp=sharing](https://drive.google.com/file/d/1BQEHqOsv9BnafEPjDYdAMcOeOxTfNlZk/view?usp=sharing)

**Comment by adrianeboyd** [comment, marked as answer]
`thinc` tries to import tensorflow if it's available, but it looks like something has gone wrong with your tensorflow installation. You don't need tensorflow for spacy or `init vectors`, so the same command in a venv without tensorflow should work fine.

**Comment by waryak** [comment]
Got the same errors after just rebuilding existing builds with fixed versions...

Yeah, i see that `tensorflow-estimator 2.6.0` implicitley was updated to 2.7.0 by `tensorflow-hub==0.9.0` or `tensorflow-text==2.6.0`, bringing problems
